### ISSUE TITLE
[v6.0.0-beta3] Error  command   fos:elastica:create --verbose  with php 8.0

### ISSUE THREAD

**Issue by devDzign** [FriendsOfSymfony/FOSElasticaBundle#1793]
Hello,

when i try to create an index  with the command   fos:elastica:create i have the error with php 8.0

config:

`fos_elastica:
    clients:
        default: { host: 'localhost', port: '9209' }
    indexes:
        book:
            properties:
                name: ~
            persistence:
                driver: orm
                model: App\Entity\Company
                provider: ~
                finder: ~`

Error  console:

`In Http.php line 178:
                                                                                                                                                         
  [Elastica\Exception\ResponseException]                                                                                                                 
  class java.util.ArrayList cannot be cast to class java.util.Map (java.util.ArrayList and java.util.Map are in module java.base of loader 'bootstrap')`

je vous met la trace complete: 

`Exception trace:
  at D:\project\symfony_projects\sf53\sf53-turbo\vendor\ruflin\elastica\src\Transport\Http.php:178
 Elastica\Transport\Http->exec() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\ruflin\elastica\src\Request.php:181
 Elastica\Request->send() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\ruflin\elastica\src\Client.php:512
 Elastica\Client->request() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\friendsofsymfony\elastica-bundle\src\Elastica\Client.php:60
 FOS\ElasticaBundle\Elastica\Client->request() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\ruflin\elastica\src\Client.php:547
 Elastica\Client->requestEndpoint() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\ruflin\elastica\src\Index.php:664
 Elastica\Index->requestEndpoint() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\ruflin\elastica\src\Index.php:431
 Elastica\Index->create() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\friendsofsymfony\elastica-bundle\src\Command\CreateCommand.php:72
 FOS\ElasticaBundle\Command\CreateCommand->execute() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\symfony\console\Command\Command.php:288
 Symfony\Component\Console\Command\Command->run() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\symfony\console\Application.php:992
 Symfony\Component\Console\Application->doRunCommand() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\symfony\framework-bundle\Console\Application.php:96
 Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\symfony\console\Application.php:291
 Symfony\Component\Console\Application->doRun() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\symfony\framework-bundle\Console\Application.php:82
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\symfony\console\Application.php:167
 Symfony\Component\Console\Application->run() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\symfony\runtime\Runner\Symfony\ConsoleApplicationRunner.php:56
 Symfony\Component\Runtime\Runner\Symfony\ConsoleApplicationRunner->run() at D:\project\symfony_projects\sf53\sf53-turbo\vendor\autoload_runtime.php:35
 require_once() at D:\project\symfony_projects\sf53\sf53-turbo\bin\console:11

fos:elastica:create [--index [INDEX]] [--no-alias]

exit status 1
`

please help me, if you knew what to do with php 8 version

**Comment by sdonmt** [comment]
I have the exact same issue. Is there any solution to this? I use it with Symfony 4.4 Might there be a conflict anyhow?

**Comment by sdonmt** [comment]
Is there any hint to this or is this project not maintained anymore?

**Comment by ga-tb** [comment]
I had this exact error when running `fos:elastica:reset`

It was happening because I was trying to publish to an elasticsearch instance running v6.5.

This branch (FOS v6.0.x) is only compatible with ES v7.x.

**Upgrading my ES instance fixed the issue for me.**

**Comment by deguif** [comment]
@ga-tb you're absolutely right as version 6.x of this bundle is only compatible with Elastica version 7.1 which is only compatible with ES version 7.x
